<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLThreeDViewsControllerWidget</class>
 <widget class="qMRMLWidget" name="qMRMLThreeDViewsControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../qMRMLWidgets.qrc">:/Icons/ViewAxis.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="PitchButton">
       <property name="toolTip">
        <string>Pitch the view of the scene.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewPitch.png</normaloff>:/Icons/ViewPitch.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="CenterButton">
       <property name="toolTip">
        <string>Center the 3D view on the scene.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCenter.png</normaloff>:/Icons/ViewCenter.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="ThreeDViewGrabButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Capture a screenshot of the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCapture.png</normaloff>:/Icons/ViewCapture.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="ZoomInButton">
       <property name="toolTip">
        <string>Zoom in on the scene by a small amount.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewZoomIn.png</normaloff>:/Icons/ViewZoomIn.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="SpinButton">
       <property name="toolTip">
        <string>Spin the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewSpin.png</normaloff>:/Icons/ViewSpin.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="RollButton">
       <property name="toolTip">
        <string>Roll the view of the scene.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewRoll.png</normaloff>:/Icons/ViewRoll.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="OrthoButton">
       <property name="toolTip">
        <string>Toggle between orthographic and perspective rendering in the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewPerspective.png</normaloff>
         <normalon>:/Icons/ViewOrtho.png</normalon>:/Icons/ViewPerspective.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="SceneSnapshotButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Capture and name a scene snapshot.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCamera.png</normaloff>:/Icons/ViewCamera.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="ZoomOutButton">
       <property name="toolTip">
        <string>Zoom out of the scene by a small amount.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewZoomOut.png</normaloff>:/Icons/ViewZoomOut.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="RockButton">
       <property name="toolTip">
        <string>Rock the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewRock.png</normaloff>:/Icons/ViewRock.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="YawButton">
       <property name="toolTip">
        <string>Yaw the view of the scene.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewYaw.png</normaloff>:/Icons/ViewYaw.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="VisibilityButton">
       <property name="toolTip">
        <string>Toggle visibility of elements in the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/VisibleOn.png</normaloff>:/Icons/VisibleOn.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="SelectSceneSnapshotMenuButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Restore or delete saved scene snapshots.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewCameraSelect.png</normaloff>:/Icons/ViewCameraSelect.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="StereoButton">
       <property name="toolTip">
        <string>Select among stereo viewing options (3DSlicer must be started with stereo enabled to use these features).</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewStereo.png</normaloff>:/Icons/ViewStereo.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionNoStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Stereo</string>
   </property>
  </action>
  <action name="actionSwitchToRedBlueStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red/blue</string>
   </property>
  </action>
  <action name="actionSwitchToAnaglyphStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anaglyph</string>
   </property>
  </action>
  <action name="actionSwitchToCrystalEyesStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CrystalEyes</string>
   </property>
  </action>
  <action name="actionSwitchToInterlacedStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interlaced</string>
   </property>
   <property name="toolTip">
    <string>Switch to Interlaced stereo mode</string>
   </property>
  </action>
  <action name="actionSet3DAxisVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D cube</string>
   </property>
  </action>
  <action name="actionSet3DAxisLabelVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D axis label</string>
   </property>
  </action>
  <action name="actionSetLightBlueBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light blue background</string>
   </property>
   <property name="toolTip">
    <string>Set light blue background</string>
   </property>
  </action>
  <action name="actionSetBlackBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black background</string>
   </property>
   <property name="toolTip">
    <string>Set black background</string>
   </property>
  </action>
  <action name="actionSetWhiteBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White background</string>
   </property>
   <property name="toolTip">
    <string>Set white background</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
